packages/utils/src/types/index.ts

```diff
@@ -17,41 +17,18 @@ export type PackedGroth16Proof = [
 ]
 
 /**
- * Supported proof types:
- * {@link @zk-kit/eddsa-proof!generate | EdDSA}
- * {@link @zk-kit/poseidon-proof!generate | Poseidon}
- * {@link https://github.com/semaphore-protocol/semaphore/tree/main/packages/proof | Semaphore}.
- * @enum
- */
-export enum Proof {
-    EDDSA = "eddsa",
-    POSEIDON = "poseidon",
-    SEMAPHORE = "semaphore"
-}
```

Comment:
don't you like the enums?
With enums I don't have to think about whether e.g I must write "eddsa" or "EdDSA" or ...

Reply:
Related: https://github.com/privacy-scaling-explorations/zk-kit/pull/275#discussion_r1580671559